docs_2.0/manual_EN/3.quick-start/3.connect-to-nebula-graph.md

```diff
@@ -0,0 +1,72 @@
+# Connect to Nebula Graph
+
+Nebula Graph supports multiple types of clients, including a CLI client, a GUI client, and clients developed in popular programming languages. This topic provides an overview of Nebula Graph clients and a basic instruction on how to use the native CLI client, nebula-console.
```

Comment:
Are CLI client and nebula-console the same? I think it's not clear here

Reply:
Yes, they are the same. The first sentence shows that Nebula Graph supports several types of client. Your point being?